### PR TITLE
SEO-172143 Windowsforms links

### PR DESCRIPTION
**Pull request by Scholar-jaruha**
Hi @MallikaRK ,
Please find the changes made.
**Note**: I have changed the folder name in the file explorer but not showing the changes in the PR.
Regards,
Scholar.

### CODE REVIEW

File: WindowsForms/GridGrouping/Data-Binding.md [hunk 1]
```diff
@@ -2183,7 +2183,7 @@ End Sub
 
 After finishing the above steps run the sample, the Grid will be loaded with the unbound column as shown in the following screenshot,
 
-![Data-Binding_img5](Data-Binding_images/Data-Binding_img5.jpeg)
+![Unbound Column in WindowsForms](data-binding_images/windowsforms-grid-grouping-unbound-column.jpeg)
```
Comment (MallikaRK) [inline WindowsForms/GridGrouping/Data-Binding.md:2186]: Control name missing in ALT Text.

File: WindowsForms/GridGrouping/Data-Binding.md [hunk 2]
```diff
@@ -264,7 +264,7 @@ Me.gridGroupingControl1.DataSource = al
 
 The Grid will be populated as follows,
 
-![Data-Binding_img1](Data-Binding_images/Data-Binding_img1.jpeg)
+![Implementation in WindowsForms](data-binding_images/windowsforms-gridgrouping-implementation.jpeg)
```
Comment (MallikaRK) [inline WindowsForms/GridGrouping/Data-Binding.md:267]: Control name is missing in ALT Text for all places. Check and add the proper control name in all places.

File: WindowsForms/GridGrouping/Getting-Started.md [hunk 3]
```diff
@@ -86,7 +86,7 @@ GridGroupingControl can be added through designer by following the below steps.
 
 2. Drag and Drop GridGroupingControl from the toolbox into the designer page. 
 
-![windows forms gridgroupingcontrol is dragged from toolbox](Getting-Started_images/Getting-Started_img1.png)
+![Windowsforms gridgroupingcontrol is dragged from toolbox](getting-started_images/windowsforms-grid-grouping-dragged-toolbox.png)
```
Comment (MallikaRK) [inline WindowsForms/GridGrouping/Getting-Started.md:89]: In Alt text, windows forms also the valid keyword. Don't change hereafter.

File: WindowsForms/GridRecordNavigationControl/Getting-Started.md [hunk 4]
```diff
@@ -40,20 +40,20 @@ You can drag both Grid Record Navigation control and Grid control (either Grid c
 
 1. Drag the GridRecordNavigationControl object from your toolbox and drop it on the form.
 
-   ![Grid-Record-Navigation-Control_images1](Creating-Grid-Record-Navigation-Control_images/Creating-Grid-Record-Navigation-Control_img1.jpeg)
+   ![Drag and Drop Toolbox in WindowsForms](creating-grid-record-navigation-control_images/windowsforms-drag-drop-toolbox.jpeg)
```
Comment (MallikaRK) [inline WindowsForms/GridRecordNavigationControl/Getting-Started.md:43]: Control name missing in image name.
image
Check and correct the changes in all places.

**Review by MallikaRK — CHANGES_REQUESTED** [review]
Commented

File: WindowsForms/GridGrouping/Data-Binding.md (hunk 2 above)
Comment (Scholar-jaruha) [reply WindowsForms/GridGrouping/Data-Binding.md:267]: added

File: WindowsForms/GridGrouping/Data-Binding.md (hunk 1 above)
Comment (Scholar-jaruha) [reply WindowsForms/GridGrouping/Data-Binding.md:2186]: added

File: WindowsForms/GridGrouping/Getting-Started.md (hunk 3 above)
Comment (Scholar-jaruha) [reply WindowsForms/GridGrouping/Getting-Started.md:89]: okay, sure

File: WindowsForms/GridRecordNavigationControl/Getting-Started.md (hunk 4 above)
Comment (Scholar-jaruha) [reply WindowsForms/GridRecordNavigationControl/Getting-Started.md]: changed

**Review by MallikaRK — APPROVED** [review]
Approved

**Review by Aishwarya-Ganesan — APPROVED** [review]
Changes are fine.

**Pull request merged**